### PR TITLE
fix: workaround for missing libaio.so.1 on noble

### PR DESCRIPTION
**Pull request by t0b3**
ubuntu 24.04 got screwed and only provides libaio.so.1t64
thus we need a workaround to get oracle binaries running as expected

see upstream bug report https://bugs.launchpad.net/ubuntu/+source/libaio/+bug/2067501

### CODE REVIEW

**Comment by t0b3** [comment]
@nyalldawson this one will fix the oracle tests

File: .docker/docker-qgis-test.sh [hunk 1]
```diff
@@ -149,7 +149,7 @@ if [ ${RUN_ORACLE:-"NO"} == "YES" ]; then
 
   echo "Wait a moment while loading Oracle database."
   COUNT=0
-  while ! echo exit | sqlplus -L SYSTEM/adminpass@$QGIS_ORACLETEST_DBNAME &> /dev/null
+  while ! echo exit | sqlplus -L SYSTEM/adminpass@$QGIS_ORACLETEST_DBNAME &> /var/log/oracle.log
```
Comment (nyalldawson) [inline .docker/docker-qgis-test.sh:152]: Can you revert these changes?

Comment (t0b3) [reply .docker/docker-qgis-test.sh]: I just thought it might become handy to have this output in case of future failures at hand... but no feelings, I just reverted the file :smile:

**Review by nyalldawson — APPROVED** [review]
